local-cluster/src/local_cluster.rs

```diff
@@ -278,10 +280,18 @@ impl LocalCluster {
                             stake,
                         ))
                     } else {
+                        required_mint_lamports += Self::required_validator_funding(*stake);
```

Comment:
working on making local cluster tests (and all tests) have fees.
This line will change as we need to add additional lamports to pay for fees.